server/scripts/remove-duplicate-locations.ts

```diff
@@ -239,7 +251,7 @@ if (require.main === module) {
               and CVS IDs, use: "--system kroger --system cvs".
             `.trim(),
           }),
-      handler: main,
+      handler: (args) => main(args),
```

Comment:
This is a little weird, but makes testing `main()` easier. Yargs’s types really want the handler to take an instance of `yargs.ArgumentsCamelCase<your_options>` or `yargs.Argv<your_options>` (and not just some compatible subset), but creating one of those when calling `main()` in a test kinda sucks. This is a convenient way to split the difference. TypeScript still makes sure things are compatible, but doesn’t make strict requirements about the exact function arguments.